### DISCUSSION TITLE
More Villagers compat

### DISCUSSION THREAD

**Discussion by ChatterBox27** [ejektaflex/Bountiful#189] [Mod Compat]
## Feature Request

Compatibility with the mod [More Villagers](https://www.curseforge.com/minecraft/mc-mods/more-villagers#c142)

As of right now the bounty structure doesn't spawn in the village with the other mod.
You could add bounties from the new professions. Though they'd need to be considered rare because the villagers sell late game stuff.
This would make bountiful better because there is more types of quest to take (such as, gather 16 honey combs and five skeleton kills). 

Thank you for your time, stay well.

**Comment by FrannDzs** [comment]
bump!
